services/markets/coinmarketcap/mapping.go

```diff
@@ -14199,5 +14199,11 @@ const Mapping = `[
         "type": "token",
         "token_id": "0xa184088a740c695E156F91f5cC086a06bb78b827",
         "id": 8387
+    },
+    {
+        "coin": 60,
+        "type": "token",
```

Comment:
can you create an issue to refactor this part to map as for later:
```
{
    "id: 123,
    "asset_id": "c60_t0x123`
}
```

not urgent, but will make things simpler for the future updates